website/templates/profile/notifications.mako

```diff
@@ -25,10 +25,19 @@
                         <div class="form-group">
 
                             <input type="checkbox"
-                                   data-bind="checked: subscribed"/>
-                            <label data-bind="text: list"></label>
+                                   data-bind="checked: subscribed" value="Open Science Framework General"/>
+                            <label data-bind="text: list[0]"></label>
```

Comment:
How much work would it be to convert this and the following from indexed items in a list to more concise key/value pairs